tracks/r/exercises/raindrops.md

```diff
@@ -0,0 +1,60 @@
+### Possible solution
+
+This solution is very intuitive and follows step-by-step the instructions in the exercise
+
+```r
+raindrops <- function(number) {
+  rain <- ""
+
+  if (number %% 3 == 0)
+    rain <- append(rain, "Pling")
+  
+  if (number %% 5 == 0)
+    rain <- append(rain, "Plang")
+  
+  if (number %% 7 == 0)
+    rain <- append(rain, "Plong")
+  
+  if (!(number %% 3 == 0 | number %% 5 == 0 | number %% 7 == 0)) # alternatively: if(rain == "")
```

Comment:
Would `||` work here as well?

Since [this exercise does _not_ have the `control_flow_conditionals` topic](https://github.com/exercism/r/blob/94a76fb9936ac0a1f53b4dd01747230de989be8b/config.json#L73-L82), I think the shorter `if(rain == "")` would be fine here as well.

Reply:
Works fine as well!
And agree, if(rain == "") is totally fine here as well